app/main/index.js

```diff
@@ -344,6 +344,18 @@ app.on('ready', () => {
 	ipcMain.on('realm-icon-changed', (event, serverURL, iconURL) => {
 		page.send('update-realm-icon', serverURL, iconURL);
 	});
+
+	// Update user idle status for each realm
+	// Set user idle if no activity for 60s
+	ipcMain.on('detect-presence', event => {
+		electron.powerMonitor.querySystemIdleState(60, idleState => {
```

Comment:
We should put the value `60` in a variable like `idleThreshhold`.

Reply:
sure, will do that.